src/PerformanceSensitiveAnalyzers/Annotations/PerformanceSensitiveAttribute.cs

```diff
@@ -15,7 +15,7 @@ namespace Roslyn.Utilities
     /// </remarks>
     [Conditional("EMIT_CODE_ANALYSIS_ATTRIBUTES")]
     [AttributeUsage(AttributeTargets.Constructor | AttributeTargets.Method | AttributeTargets.Property | AttributeTargets.Field, AllowMultiple = true, Inherited = false)]
-    internal sealed class PerformanceSensitiveAttribute : Attribute
+    public sealed class PerformanceSensitiveAttribute : Attribute
```

Comment:
can this be Microsoft.CodeAnalysis and not Roslyn.Utilities?

Reply:
I think that makes sense. @sharwell thoughts?